cmd/oci-image-validate/main.go

```diff
@@ -149,6 +150,19 @@ func (v *validateCmd) validatePath(name string) error {
 
 	switch typ {
 	case image.TypeManifest:
+		fv, err := os.Open(name)
```

Comment:
I don't think we need to open the file again.  Can't we put all the manifest validation behind a single function that takes a `ReadCloser`?